Makefile

```
@@ -173,7 +173,7 @@ delete-trino:
 	@$(PREFIX) rm -rf $(TOPDIR)/.trino/trino/*
 
 delete-trino-data:
-	@$(PREFIX) rm -rf $(TOPDIR)/.trino/parquet_data/{*,.minio*}
+	@$(PREFIX) rm -rf $(TOPDIR)/.trino/parquet_data/{*,.minio*}/**
```

Let's just change this to deleting everything in the bucket directory.
```suggestion
	@$(PREFIX) rm -rf $(TOPDIR)/.trino/parquet_data/koku-bucket/*
```